src/license/license.cpp

```diff
@@ -177,9 +177,10 @@ void LicenseChecker::RevalidateLicense(const std::string &license_key, const std
   }
 }
 
-void LicenseChecker::EnableTesting() {
+void LicenseChecker::EnableTesting(const LicenseType license_type) {
   enterprise_enabled_ = true;
   is_valid_.store(true, std::memory_order_relaxed);
+  license_type_ = license_type;
```

Comment:
You will hate me, but the log on the next line is super misleading if the license type is OEM. Could you please rephrase that? Nothing fancy is necessary, just generalize it: "The specified license type is set for testing", or something something that is true for  both cases.